src/oss/paths/authorizations_authID_password.yml

```diff
@@ -0,0 +1,25 @@
+patch:
```

Comment:
I'd rename this file to `legacy_authorizations_authID_password.yml` to match the suggestions above